### ISSUE TITLE
Rethink data variable prefixes

### ISSUE THREAD

**Issue by jonpalmisc** [Vector35/workflow_objc#32]
Data variable prefixes (e.g. `cl_`, `sr_`, etc.) currently have a few problems:

- They are hardcoded
- They are only two characters and some natural collisions arise
- They are not all very descriptive

The short prefixes were originally chosen to prevent bloating the name of the variable and contributing excessive width to lines in which they are used one or more times.

A couple things should be done:

- Pick better prefixes (maybe copy IDA or do something in between)
- Make prefixes constants in the code to prevent accidental collisions or de-syncs

**Comment by bb010g** [comment]
I started changing a few in #34 because I was running out of letters. I think reserving `*r_` for references is a good start. I also like reserving `*l_` for lists; `*ll_` should be a list of lists. The current naming has `mt_` for a method list entry, is this "method table"? Making a table of types (and relevant members), their runtime type names from headers / what we call them in our declared system types, and then listing the current data variable prefixes would probably be a good start for this.

**Comment by rickmark** [comment]
Maybe a better fix would be "symbol namespaces or groups"  - it would be great if you could define these as "(objc, sl_*)" to keep them away from "normal symbols"

**Comment by rickmark** [comment]
Looks like some common symbol prefixes look like:

```
_OBJC_EHTYPE_$_SomeClass

_OBJC_CLASS_$_SomeClass
_OBJC_METACLASS_$_SomeClass
__OBJC_CLASS_RO_$___SomeClass
__OBJC_METACLASS_RO_$___SomeClass

__OBJC_$_CLASS_METHODS__SomeClass
__OBJC_$_INSTANCE_METHODS_SomeClass(SomeListOfThings)
__OBJC_CLASS_PROTOCOLS_$_SomeClass

_OBJC_IVAR_$_SomeClass._someField

__OBJC_LABEL_PROTOCOL_$_SomeClass
__OBJC_PROTOCOL_$_SomeProtocol
__OBJC_$_PROTOCOL_INSTANCE_METHODS_SomeProtocol
__OBJC_$_PROTOCOL_METHOD_TYPES_SomeProtocol
__OBJC_$_PROTOCOL_REFS_SomeProtocol

__OBJC_$_CATEGORY_SomeClass_$_SomeCategory
__OBJC_$_CATEGORY_CLASS_METHODS_SomeClass_$_SomeCategory
__OBJC_$_CATEGORY_INSTANCE_METHODS_SomeClass_$_SomeCategory


__OBJC_$_PROP_LIST_SomeClass


```

**Comment by 0cyn** [comment]
Resolved in **dev builds >= 4.1.5465**
